### DISCUSSION TITLE
How to set a list variables in github action

### DISCUSSION THREAD

**Discussion by hocinemala** [community/community#36073] [Actions]
Hello,

How to set a list variables in github action?

env:
   var_list: ["1", "2", "3", "4"]

Regards

**Comment by antonblr** [comment]
The environment variables (`env:`) are actually strings -> you will have to process that string somehow in order to get the list out of it.

i.e. you can do something like that with shell:
```shell
MY_VAR="1 2 3 4 5"

for i in $MY_VAR; do
  echo $i
done
```
What is your use case? Perhaps there could be other options.
